backend/app/services/subtask.py

```
@@ -368,5 +370,207 @@ def get_new_messages_since(
 
         return messages
 
+    def delete_subtasks_from(
+        self,
+        db: Session,
+        *,
+        task_id: int,
+        from_message_id: int,
+        user_id: int,
```

Ruff (0.14.10)

Unused method argument: `user_id`

(ARG002)